job-dsls/jobs/kie_release_jobs.groovy

```diff
@@ -6,6 +6,7 @@ def dashbuilderVersion="0.9.x"
 def javadk="jdk1.8"
 def jaydekay="JDK1_8"
 def mvn="APACHE_MAVEN_3_3_9"
+def mvnVersionTest="apache-maven-3.3.9"
```

Comment:
because this is version apache-maven-3.3.9 is only required in the additional tests - if you don't like this variable and I should name it otherwise please tell me